osf/models/preprint_provider.py

```diff
@@ -6,12 +6,22 @@
 from osf.models.base import BaseModel, ObjectIDMixin
 from osf.models.licenses import NodeLicense
 from osf.models.subject import Subject
+from osf.models.user import Email, SocialAccount
 from osf.utils.datetime_aware_jsonfield import DateTimeAwareJSONField
 from osf.utils.fields import EncryptedTextField
 
 from website.util import api_v2_url
 
 
+class PreprintProviderLink(ObjectIDMixin, BaseModel):
```

Comment:
This doesn't seem specific to `PreprintProvider` in anything but name. Consider renaming to, e.g. `ExternalLink`, or creating a base class to make this more easily extensible for other places where we have links, such as `User.social.profileWebsites`